mockk/src/main/kotlin/io/mockk/impl/annotations/JvmMockInitializer.kt

```diff
@@ -55,6 +58,17 @@ class JvmMockInitializer(val gateway: MockKGateway) : MockKGateway.MockInitializ
                     arrayOf()
                 )
             }
+
+            if (property.findAnnotation<InjectMockKs>() != null) {
+                val injectMockProperty = property as KProperty1<Any, Any>
+                injectMockProperties.add(injectMockProperty)
+            }
+        }
+
+        if (!injectMockProperties.isEmpty()) {
+            for (injectMockProperty in injectMockProperties) {
+                matchInjectMockMembers(target, injectMockProperty)
```

Comment:
Naming it `injectMockKs` is okay as well, but up to you. 

From my point of view it is important to keep style at a high level of readability, maybe not all current code is following that, but that means we should make it even better than it's now. Each name should be semantically correct ideally. I prefer small names for small scopes and big names for bigger scopes.